stories/components/footer/footer.html.twig

```diff
@@ -3,16 +3,20 @@
     <div class="bix-site-footer__columns">
       <div class="bix-site-footer__column bix-site-footer__column--logo">
         <div class="bix-site-footer__logo">
-          <!-- @TODO Get a properly sized image. -->
-          <img role="presentational" aria-hidden="true" decoding="async" loading="lazy" width="150px"
-            alt="Bixal Logo letter B silhouette around a photo of a smiling child."
+          {# @TODO: Get a properly sized image, ex: 100px wide. #}
+          <img
+            role="presentational"
+            aria-hidden="true"
+            decoding="async"
+            loading="lazy"
+            width="100px"
             src="https://bixal.com/static/8fb0fe7a56370262b76be5bda8d68737/d409d/img-b-footer.png">
```

Comment:
On live it's using a 500px wide image, but is never used. We should resize to `100px` for regular screens and `200px` for HiDPI to improve performance.